src/frontend/js/hooks/usePendingContractCount/index.ts

```diff
@@ -0,0 +1,57 @@
+import { useMemo } from 'react';
+import useContractAbilities from 'hooks/useContractAbilities';
+import { useOrganizationContracts } from 'hooks/useContracts';
+import { useCourseProductRelation } from 'hooks/useCourseProductRelation';
+import { PER_PAGE } from 'settings';
+import { ContractState, CourseProductRelation, Organization } from 'types/Joanie';
+import { ContractActions } from 'utils/AbilitiesHelper/types';
+
+interface UsePendingContractCountProps {
+  organizationId?: Organization['id'];
+  courseProductRelationId?: CourseProductRelation['id'];
+}
+
+const useTeacherPendingContracts = ({
```

Comment:
I'll rename this hook `useTeacherPendingContractsCount` to explicit its purpose.